apps/hash-ai-worker-ts/src/activities/shared/get-llm-response.ts

```diff
@@ -460,11 +461,12 @@ const getOpenAiResponse = async (
     logger.debug(`OpenAI response: ${stringify(openAiResponse)}`);
   } catch (error) {
     logger.error(`OpenAI API error: ${stringify(error)}`);
-    logger.debug(`OpenAI API request payload: ${stringify(completionPayload)}`);
-    logger.debug(`Original messages: ${stringify(messages)}`);
```

Comment:
Is this now covered elsewhere?

Reply:
I added these whilst debugging something in a previous PR, but found it quite noisy when encountering other errors. I can add this back if you think it'd be useful.

Reply:
They can be left out, we can add them back if necessary.

If there are things which are useful for local development but can be noisy we can always give them a `.silly` level so we can turn them on as needed.